src/ui/pages/Networks/README.md

```diff
@@ -11,21 +11,19 @@
 
 ### Chain, chainId and external_id
 
-- `chain` is used as a slug-like unique `id`
-- `external_id` is a name for `chainId`
-- `evm_id` is an int representation of `external_id`, therefore is unnecessary
-- for chain configs coming from dApps, `chain` is generated using `nanoid()`
-- for manually added networks, `chain` is currently created equal to `external_id`. This is wrong, because `external_id` can be changed later by the user, but `chain` will remain untouched. If a user manually creates network A with external_id: `'a'`, then later updates external_id to `'b'`, then later creates network `AA`, sets its external_id to `'a'`, network `AA` will overwrite network `A`.
+- HEX chain id is stored in `specification` object inside NetworkConfig
+- for chain configs coming from dApps, `id` is get from backend for the chain with the same `chainId` or directly from `chainId`
+- for manually added networks, `id` is currently created equal to `chainId` and will be updated if `chainId` is updated.
```

Comment:
Uhm, I'm not sure I see this? I think we always derive `id` from `chainId` using `toCustomNetworkId` function. Are there exceptions?

Reply:
You are correct. There is a mistake in readme.